code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
https://puu.sh/uGaiC/87422e24a1.png
At this point, I'm pretty much willing to try anything.